bridge-deploy/bridge_deploy/cli.py

```diff
@@ -0,0 +1,276 @@
+import click
+
+# from web3 import Web3, EthereumTesterProvider
+
+from deploy_tools.cli import (
+    jsonrpc_option,
+    keystore_option,
+    gas_option,
+    gas_price_option,
+    nonce_option,
+    auto_nonce_option,
+    connect_to_json_rpc,
+    retrieve_private_key,
+    get_nonce,
+)
+from deploy_tools.deploy import build_transaction_options
+from deploy_tools.files import validate_and_format_address, InvalidAddressException
+
+from bridge_deploy.home import (
+    deploy_home_bridge_contract,
+    initialize_home_bridge_contract,
+)
+
+
+def validate_address(ctx, param, value):
+    # TODO:
+    # Potentially reformat this to deploy-tools? (Copypasta from validator-set-deploy)
+    # Refactoring might be a thing.
+    """This function must be at the top of click commands using it"""
+    try:
+        return validate_and_format_address(value)
+    except InvalidAddressException as e:
+        raise click.BadParameter(
+            f"The address parameter is not recognized to be an address: {value}"
+        ) from e
+
+
+validator_set_address_option = click.option(
+    "--validator-set-address",
+    help='The address of the validator set contract, "0x" prefixed string',
+    type=str,
+    required=True,
+    callback=validate_address,
+    metavar="VALIDATOR_SET_ADDRESS",
+    envvar="VALIDATOR_SET_ADDRESS",
+)
+
+block_reward_address_option = click.option(
+    "--block-reward-address",
+    help='The address of the block reward contract, "0x" prefixed string',
+    type=str,
+    required=True,
+    callback=validate_address,
+    metavar="BLOCK_REWARD_ADDRESS",
+    envvar="BLOCK_REWARD_ADDRESS",
+)
+
+owner_address_option = click.option(
```

Comment:
What is the owner's power for the bridge? Shouldn't we use `0x00...01` since we cannot have governance of the bridge (and 0x00...0 is not allowed).

Reply:
Agree that we don't need the option. I think `0` is only not allowed during initialization, we can still set it afterwards. Would prefer having a temporary owner and setting it to `0` later over setting it to `1` during initialization -- mostly for aesthetic purposes.

Reply:
That way we could never change anything regarding the bridge later on. This could also be a MultiSigWallet?

Reply:
Only safe option would be to have a multisig consisting of the validators. I guess we could implement this in the validator proxy, but it would take some effort. So I think we should just pick some good defaults, keep them static, and update it by forking if necessary (I guess this requires the update-contract-state-in-fork feature in Parity though).

Reply:
setOwner is internal, but the method we should actually use is transferProxyOwnership anyway, which again checks for address(0). Resorting to setting the owner to address(1) now.